src/main/java/net/william278/velocitab/commands/VelocitabCommand.java

```diff
@@ -0,0 +1,67 @@
+package net.william278.velocitab.commands;
+
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.command.SimpleCommand;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.TextColor;
+import net.william278.desertwell.AboutMenu;
+import net.william278.desertwell.Version;
+import net.william278.velocitab.Velocitab;
+
+import java.util.List;
+
+public class VelocitabCommand implements SimpleCommand {
+    private final AboutMenu aboutMenu;
+    private final Velocitab plugin;
+
+    public VelocitabCommand(Velocitab plugin) {
+        this.plugin = plugin;
+        aboutMenu = AboutMenu.create("Velocitab")
+                .withDescription("Velocitab is a super-simple Velocity TAB menu plugin that uses scoreboard team client-bound packets to actually sort player lists without the need for a backend plugin.")
+                .withVersion(Version.fromString(plugin.getVersion(), "-"))
+                .addAttribution("Author",
+                        AboutMenu.Credit.of("William278").withDescription("Click to visit website").withUrl("https://william278.net"))
+                .addAttribution("Contributors",
+                        AboutMenu.Credit.of("Ironboundred").withDescription("Coding"),
+                        AboutMenu.Credit.of("Emibergo02").withDescription("Coding"))
+                .addButtons(
+                        AboutMenu.Link.of("https://william278.net/docs/velocitab").withText(" Wiki").withIcon("⛏"),
+                        AboutMenu.Link.of("https://discord.gg/tVYhJfyDWG").withText(" Discord").withIcon("⭐").withColor("#6773f5"),
+                        AboutMenu.Link.of("https://modrinth.com/plugin/velocitab").withText(" Modrinth").withIcon("X").withColor("#589143"));
```

Comment:
```suggestion
                        AboutMenu.Link.of("https://william278.net/docs/velocitab").withText("Docs").withIcon("⛏"),
                        AboutMenu.Link.of("https://discord.gg/tVYhJfyDWG").withText("Discord").withIcon("⭐").withColor("#6773f5"),
                        AboutMenu.Link.of("https://modrinth.com/plugin/velocitab").withText("Modrinth").withIcon("X").withColor("#589143"));
```
nit: DesertWell will already add a space between the icon and the text, so no need for a space -- and change do "Docs" for consistency with other plugins (and the nomenclature on my site)